projects/pastanaga/src/lib/expand/expand.component.ts

```diff
@@ -19,6 +19,7 @@ const EXPANSION_PANEL_ANIMATION_TIMING = '225ms cubic-bezier(0.4,0.0,0.2,1)';
 export class ExpandComponent implements OnInit {
     @Input() toggleTooltip: string[] = ['', ''];
     @Input() openOnInit = false;
+    @Input() maintainContentOnClose = false;
```

Comment:
would be better to use a setter with `coerceBooleanProperty`

Reply:
done